mlrun/runtimes/base.py

```diff
@@ -1311,14 +1327,16 @@ def _get_object_label_selector(object_id: str) -> str:
         """
         pass
 
-    @staticmethod
-    @abstractmethod
-    def _get_possible_mlrun_class_label_values() -> List[str]:
+    def _get_possible_mlrun_class_label_values(
+        self, class_mode: typing.Union[RuntimeClassMode, str] = None
+    ) -> List[str]:
         """
         Should return the possible values of the mlrun/class label for runtime resources that are of this runtime
         handler kind
         """
-        pass
+        if not class_mode:
+            return list(self.class_modes.values())
+        return [self.class_modes.get(class_mode, [])]
```

Comment:
This will either return `[[]]` or `["mode-name"]`.
Don't think there's a nice magic syntactic sugar way to get what you want.
```suggestion
        class_mode_name = self.class_modes.get(class_mode, None)
        return [class_mode_name] if class_mode_name else []
```
Or you can use some python3.9 features which can make it more readable than the previous suggestion:
```suggestion
        if class_mode_name := self.class_modes.get(class_mode, None):
            return [class_mode_name]
        return []
```